tests/models.py

```diff
@@ -6,13 +6,25 @@
 class NaiveDateTimeTestModel(models.Model):
     aware = models.DateTimeField()
     naive = NaiveDateTimeField()
+    timezone = models.CharField(max_length=100, blank=True, default="utc")
```

Comment:
Same, is there a constant we can use? If not, should we define our own and make it "UTC" for consistency?

Reply:
This one really is a one-off for the test model - I think it's OK for this not to be a constant.